src/dynamic_fee.rs

```diff
@@ -56,24 +66,42 @@ impl Miner {
                     .await
                     .unwrap();
 
-                let calculated_fee = match strategy.as_str() {
-                    "helius" => response["result"]["priorityFeeEstimate"]
-                        .as_f64()
-                        .map(|fee| fee as u64)
-                        .ok_or_else(|| {
-                            format!("Failed to parse priority fee. Response: {:?}", response)
-                        })
-                        .unwrap(),
-                    "triton" => response["result"]
-                        .as_array()
-                        .and_then(|arr| arr.last())
-                        .and_then(|last| last["prioritizationFee"].as_u64())
-                        .ok_or_else(|| {
-                            format!("Failed to parse priority fee. Response: {:?}", response)
-                        })
-                        .unwrap(),
-                    _ => return self.priority_fee.unwrap_or(0),
-                };
+                let calculated_fee =
+                    match strategy.as_str() {
+                        "helius" => response["result"]["priorityFeeEstimate"]
+                            .as_f64()
+                            .map(|fee| fee as u64)
+                            .ok_or_else(|| {
+                                format!("Failed to parse priority fee. Response: {:?}", response)
+                            })
+                            .unwrap(),
+                        "triton" => response["result"]
```

Comment:
We can probably combine the `triton` and `alchemy` ones here, yeah?

Reply:
If we pass the percentile parameter, I believe triton does the averaging for us before returning it so the implementations don't match. If alchemy adds the percentile parameter (which I did try to use and got an error) then we could combine them. Or I suppose we could combine them if we dropped the percentile param out of the triton request.

Reply:
Ah, that makes sense!